docs/book/src/development/development.md

```diff
@@ -342,6 +342,86 @@ $ sudo ln -s /run/user/$(id -u)/podman/podman.sock /var/run/docker.sock
 [hack-ci-gce-project]: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/gce-project.sh
 [hack-ci-openstack]: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/main/hack/ci/openstack.sh
 
+## Cluster Creation with ClusterClass
+
+This section provides a guide to create a Kubernetes cluster using ClusterClass. It covers the process of setting up environment variables, creating a Kind cluster, configuring a secret, applying ClusterClass and deploying the cluster.
+
+### Setting Environment Variables
+
```

Comment:
Good point. The main difference and benefit here is that we are adding a ClusterClass that is geared specifically at development. It has defaults that fit the devstack setup so that as few variables as possible are needed. I think we need to point this out @maxrantil 
I'll add a few suggestions

Reply:
I tried to explain it more clear in the documentation now. See what you think now.